bin/blazegraph_load_api.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# On a debian package installation, the original file could be found at:
+#/usr/bin/loadRestAPI.sh
+
+if [ "$1" == "" -o "$1" == "-h" -o "$1" == "--help" ]; then
+    echo "Usage: $0 {file_or_directory} [{graph_namespace}]"
+    exit
+else
+    FILE_OR_DIR=$1
+fi
+
+if [ ! -z "$2" ]; then
+    NAMESPACE=$2
+else
+    NAMESPACE=kb
+fi
+
+[ -f /etc/default/blazegraph ] && source /etc/default/blazegraph
+[ -z "$JETTY_PORT" ] && JETTY_PORT=9999
+
+BG_URL="http://localhost:${JETTY_PORT}/blazegraph/dataloader"
+
+# On a debian package installation, there is a file at
+# /etc/blazegraph/RWStore.properties
+# That file is almost identical to the source file in 
+# BLAZEGRAPH_RELEASE_2_1_4/src/resources/deployment/nss/WEB-INF/RWStore.properties
+# Other installations might put this file in:
+#export NSS_DATALOAD_PROPERTIES=/usr/local/blazegraph/conf/RWStore.properties
+
+[ -z "${NSS_PROPERTIES}" ] && NSS_PROPERTIES=/etc/blazegraph/RWStore.properties
```

Comment:
This is because blazegraph can be installed in different ways.  This file works on my laptop.  It's not working yet on the blazegraph-dev box and when I manually edited it on that box to get it running, it did not work.  I found the relevant files in the tomcat webapp.  For now, this script is using generic code from the blazegraph repo itself.  But, anyway, this PR has generic code that uses common defaults, not the deployment-system specific stuff.  Because there are different ways to deploy blazegraph, we may need copies of scripts that work for each deployment style.  FYI, this file came from https://github.com/blazegraph/database/blob/master/blazegraph-deb/src/deb/bin/loadRestAPI.sh